docs/content/how-to/baremetal-none.md

```diff
@@ -0,0 +1,429 @@
+community-operators-catalog-6f5797cdc4-2hlcp      1/1     Running    0               6m29s
+control-plane-operator-749b94cf54-p2lg2           1/1     Running    0               7m23s
+etcd-0                                            1/1     Running    0               6m46s
+hosted-cluster-config-operator-6646d8f868-h9r2w   0/1     Running    0               6m34s
+ignition-server-7797c5f7-vkb2b                    1/1     Running    0               7m20s
+ingress-operator-5dc47b99b7-jttpg                 0/2     Init:0/1   0               6m35s
+konnectivity-agent-85f979fcb4-67c5h               1/1     Running    0               6m45s
+konnectivity-server-576dc7b8b7-rxgms              1/1     Running    0               6m46s
+kube-apiserver-66d99fd9fb-dvslc                   2/2     Running    0               6m43s
+kube-controller-manager-68dd9fb75f-mgd22          1/1     Running    0               6m42s
+kube-scheduler-748d9f5bcb-mlk52                   0/1     Running    0               6m42s
+machine-approver-c8c68ffb9-psc6n                  0/1     Running    0               6m11s
+oauth-openshift-7fc7dc9c66-fg258                  1/1     Running    0               6m8s
+olm-operator-54d7d78b89-f9dng                     2/2     Running    0               6m22s
+openshift-apiserver-64b4669d54-ffpw2              2/2     Running    0               6m41s
+openshift-controller-manager-7847ddf4fb-x5659     1/1     Running    0               6m38s
+openshift-oauth-apiserver-554c449b8f-lk97w        1/1     Running    0               6m41s
+packageserver-6fd9f8479-pbvzl                     0/2     Init:0/1   0               6m22s
+redhat-marketplace-catalog-8cc88f5cb-hbxv9        1/1     Running    0               6m29s
+redhat-operators-catalog-b749d6945-2bx8k          1/1     Running    0               6m29s
+~~~
+
+The hosted cluster's kubeconfig can be extracted as:
+
+~~~sh
+oc extract -n ${CLUSTERS_NAMESPACE} secret/${HOSTED}-admin-kubeconfig --to=- > ${HOSTED}-kubeconfig
+oc get clusterversion --kubeconfig=${HOSTED}-kubeconfig
+~~~
+
+### Adding a bare metal worker
+
+* Download the RHCOS live ISO into the httpd server (for example into `/var/www/html/hypershift-none/` on an apache server hosted at www.example.com)
+
+~~~sh
+mkdir -p /var/www/html/hypershift-none/
+curl -s -o /var/www/html/hypershift-none/rhcos-live.x86_64.iso https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.9/4.9.0/rhcos-live.x86_64.iso
+~~~
+
+* Download the ignition generated in the hostedcluster
+
+~~~sh
+curl -s -k -H "Authorization: Bearer $(oc -n clusters-${HOSTED} get secret $(oc -n clusters-${HOSTED} get secret | grep token-${HOSTED}  | awk '{print $1}') -o jsonpath={.data.token})" https://$(oc get node -o wide | grep master | head -1 | awk '{print $6}'):$(oc -n clusters-${HOSTED} get svc ignition-server -o jsonpath={.spec.ports[0].nodePort})/ignition > /var/www/html/hypershift-none/worker.ign
```

Comment:
IMO this would be easier to understand if it was split into several lines like [this](https://github.com/hardys/dev-scripts/blob/hypershift_wip/hypershift/03_hypershift_nodes.sh#L18..L22) vs the one-liner with subshell

Reply:
Thanks for the comment Steven. I've modified it, would you mind to take a look again?